src/.dockerignore

```diff
@@ -0,0 +1,9 @@
+# node modules found in any project
+**/node_modules
+
+rust/target
+
+js/grapl-cdk/zips
+js/grapl-cdk/cdk.out
+
+js/engagement_view/build
```

Comment:
It'd probably be good to add some more things from `.gitignore`, like `__pycache__`, `dist`, egg stuff, etc.

Reply:
It looks to me like a lot of the items in `.gitignore` are no longer relevant. Like everything under `# Distribution / packaging`, as all of that is done in Docker. `dist` was used by the dobi system, so that could probably be removed from `.gitignore`, but even still that's a level up from the dir we're using for Docker context.

Reply:
Nothing in `.gitignore` stands out to me as relevant for `src/.dockerignore`, but maybe I'm missing something.

@christophermaier How about I make a ticket to revisit `.gitignore` `src/.dockerignore` and in a separate PR?

Reply:
@inickles-grapl Yeah, that's fine 👍